flink-core/src/main/java/org/apache/flink/configuration/DelegatingConfiguration.java

```diff
@@ -178,14 +178,19 @@ public String toString() {
 
 	@Override
 	public Set<String> keySet() {
+
```

Comment:
We usually don't have an empty line when a method block starts (also applies to lines 185 and in `DelegatingConfigurationTest` lines 96 and 110).